src/Altinn.App.Api/Controllers/ProcessController.cs

```diff
@@ -227,42 +245,55 @@ private async Task<bool> CanTaskBeEnded(Instance instance, string currentTaskId,
             [FromRoute] int instanceOwnerPartyId,
             [FromRoute] Guid instanceGuid,
             [FromQuery] string? elementId = null,
-            [FromQuery] string? language = null)
+            [FromQuery] string? language = null,
+            [FromBody] ProcessNext? processNext = null)
         {
             try
             {
-                ProcessNext? processNext = null;
-                if (Request.Body != null && Request.Body.CanRead)
-                {
-                    processNext = await DeserializeFromStream<ProcessNext>(Request.Body);
-                }
-
                 Instance instance = await _instanceClient.GetInstance(app, org, instanceOwnerPartyId, instanceGuid);
 
-                if (instance?.Process == null)
+                var currentTaskId = instance.Process.CurrentTask?.ElementId;
```

Comment:
Super nitpicky, can ignore: Use explicit datatype when you can't tell the type 100% from the right side.

Reply:
There seems to be a style-rule that matches that expectation. 

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008#csharp_style_var_when_type_is_apparent